### PR TITLE
Add some desktop niceties for xwordgrinder

### LINKED ISSUE #141: Desktop file?
Over the weekend or sometime in the very near future, I'd planned on adding a `.desktop` file for the Fedora xwordgrinder package so that it appears in desktop environment menus. Is that something you'd like me to submit upstream and if so, where's a good place to put it? I was thinking `extras/` alongside the icon file, but if you have somewhere else in mind I'm happy to put it somewhere else.

### PR DESCRIPTION
**Pull request by funnelfiasco**
Fixes #141

### CODE REVIEW

File: extras/wordgrinder.mime [hunk 1]
```diff
@@ -0,0 +1,2 @@
+application/wordgrinder
```
Comment (davidgiven) [inline extras/wordgrinder.mime:1]: Should probably be `application/x-wordgrinder` --- I'm not registered with IANA...

File: extras/wordgrinder.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=WordGrinder
+Comment=Word processor for processing words
+Exec=xwordgrinder %f
+Icon=wordgrinder
+MimeType=application-wordgrinder
```
Comment (davidgiven) [inline extras/wordgrinder.desktop:6]: As below (also, is the hyphen intentional?)

**Review by davidgiven — CHANGES_REQUESTED** [review]

**Comment by funnelfiasco** [comment]
Squashed commit pushed to address requested changes

**Review by davidgiven — APPROVED** [review]

**Comment by davidgiven** [comment]
Thanks very much!

**Pull request merged**